ocs_ci/deployment/deployment.py

```diff
@@ -241,6 +242,95 @@ def do_deploy_rdr(self):
             deploy_dr = MultiClusterDROperatorsDeploy(dr_conf)
             deploy_dr.deploy()
 
+    def do_deploy_lvmo(self):
+        """
+        call lvm deploy
+
+        """
+        if not config.DEPLOYMENT["install_lvmo"]:
+            logger.warning("LVMO deployment will be skipped")
+            return
+        vsphere = VSPHERE(
```

Comment:
you should not have platform specific code called from within a method of the base deployment class.
I suggest you create a generic function, which is common to all platforms, where this platform-specific implementation will reside. You can have only one or some of those methods actually implemented and the rest as interfaces

Reply:
I understand. deploy_ocs is common for all platform but mine the disk adding can be changed from platform to platform although we are using only one platform for now. I'll make a if statement but for now it will handle only vsphere.

Reply:
Moved disk adding which is the only difference between platform to sno deployment. This way all deployment of lvmo will be generic for all platform.